bci_tester/data.py

```diff
@@ -28,6 +29,14 @@
 #: places
 OS_VERSION = os.getenv("OS_VERSION", "15.4")
 
```

Comment:
add ciomment that the last entry is the special and hence the list needs to be sorted, or sort it explicitly..